src/main.rs

```diff
@@ -93,16 +87,13 @@ async fn store_file(filename: &str, bytes: Bytes, args: &Args) -> std::io::Resul
     Ok(())
 }
 
-async fn accept_form(
-    ContentLengthLimit(mut multipart): LimitedMultipart,
-    Extension(args): Extension<Arc<Args>>,
-) -> Redirect {
+async fn accept_form(Extension(args): Extension<Arc<Args>>, mut multipart: Multipart) -> Redirect {
```

Comment:
See https://github.com/tokio-rs/axum/blob/main/axum/CHANGELOG.md#060-25-november-2022

breaking: ContentLengthLimit has been removed. Use DefaultBodyLimit instead (https://github.com/tokio-rs/axum/pull/1400)

We also needed to switch the order of the extractors: https://docs.rs/axum/0.6.18/axum/extract/index.html#the-order-of-extractors